### PR TITLE
Create LIT6934Edification

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a LIT record for a composition in BLorient8823 as recommended by Dorothea in https://github.com/BetaMasaheft/Documentation/issues/2482.

### CODE REVIEW

File: new/LIT6934Edification.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6934Edification" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Composition for moral edification</title>
```
Comment (eu-genia) [inline new/LIT6934Edification.xml:8]: en

**Review by eu-genia — CHANGES_REQUESTED** [review]
very nice!

File: new/LIT6934Edification.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Composition for moral edification in three parts.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="Miscellanea"/>
+                    <term key="Philosophy"/>
```
Comment (thea-m) [inline new/LIT6934Edification.xml:38]: I would say also ChristianLiterature

**Review by thea-m — CHANGES_REQUESTED** [review]
Very nice

**Pull request merged**